src/app/dashboard/statistics/page.tsx

```
@@ -17,6 +20,112 @@ import { formatDateToJSTWithSlash } from "@/utils/dateUtils";
+function useGraphExport() {
+  // 非破壊的エクスポート: oncloneでスタイル調整
+  const exportGraphAsImage = async (
+    elementRef: React.RefObject<HTMLElement | HTMLDivElement | null>,
+    filename: string = "statistics-charts"
+  ) => {
+    if (!elementRef.current) return;
+
+    try {
+      // 元のDOMは一切変更せず、oncloneでクローンを調整
+      const canvas = await html2canvas(elementRef.current, {
+        backgroundColor: '#ffffff',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)